### PR TITLE
MSH.9 Implementation (Message Type) #726

### PR DESCRIPTION
**Pull request by cunnia4**
Jason and I discussed hardcoding in the message type values, as these probably wouldn't be available in the native FHIR object and are most likely only in the expected FHIR object in this repo as a product of the original HL7 conversion. Different messages can be implemented the same way if support for those is needed.

### CODE REVIEW

File: data/Templates/FhirToHL7v2/ADT_A08.liquid [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {% comment %} {{ '' | debug }} {% endcomment %}
 {%- assign messageHeaderObject = fhirData.entry | select: 'resource.resourceType', 'MessageHeader' | get_index: 0 -%}
+{% comment %} Since this is the ADT_A08 liquid template, the following 3 can be hardcoded {% endcomment %}
```
Comment (jakiefermsft) [inline data/Templates/FhirToHL7v2/ADT_A08.liquid:3]: I _think_ this comment might be the culprit of your extra newline.

Comment (cunnia4) [reply data/Templates/FhirToHL7v2/ADT_A08.liquid:3]: Shoot. It is, should have tested after adding the comment. I'll just get rid of it.

Comment (jasonheld-msft) [reply data/Templates/FhirToHL7v2/ADT_A08.liquid:3]: Your other option is to add the dashes, similar to some of the other ones:

{%- -%}

Comment (jasonheld-msft) [reply data/Templates/FhirToHL7v2/ADT_A08.liquid]: I agree @ accidentally

Comment (cunnia4) [reply data/Templates/FhirToHL7v2/ADT_A08.liquid]: thanks guys, that worked!

**Review by destinyhillis — APPROVED** [review]

**Pull request merged**